libs/dh/charges/feature-prices/src/lib/drawer/dh-charges-prices-drawer.component.ts

```diff
@@ -46,6 +52,7 @@ export class DhChargesPricesDrawerComponent {
   openDrawer(charge: ChargeV1Dto) {
     this.charge = charge;
     this.drawer.open();
+    this.chargesMessageTabComponent.loadMessages();
```

Comment:
Maybe this should be removed as the component is feature flagged.

Reply:
Good point, but then no dummy data is added to the list during development/on development environment. I will split the message in two, one handling the desired api-call for messages, and another providing dummy-data.

Reply:
I took a step further, using the feature flag to switch message archive api calls on/off depending on the environment.